src/embed/connection.rs

```diff
@@ -671,13 +671,13 @@ async fn do_push<'a, 'b>(
 
     let batch_push_info =
         sync::push(&sync_id, ctx.store, ctx.lc, ctx.client_id, &pusher, req).await?;
-    Ok(sync::PushResponse { batch_push_info })
+    Ok(sync::TryPushResponse { batch_push_info })
 }
 
-async fn do_begin_pull<'a, 'b>(
+async fn do_try_begin_pull<'a, 'b>(
```

Comment:
the try is in two different places in the name for pull: do_maybe_end_TRY_pull and do_TRY_begin_pull. since i think the name of the thing is "try pull" it should be do_begin_try_pull and do_maybe_end_try_pull. i realize it is a mouthful.

Reply:
ha!, I was going to make it TryX but I failed to consistently rename this